homeassistant/components/here_travel_time/sensor.py

```diff
@@ -115,6 +118,61 @@
 )
 
 
+def sensor_descriptions(travel_mode: str) -> tuple[SensorEntityDescription, ...]:
+    """Construct SensorEntityDescriptions."""
+    return (
+        SensorEntityDescription(
+            name="Duration",
+            icon=ICONS.get(travel_mode, ICON_CAR),
+            key=ATTR_DURATION,
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement=TIME_MINUTES,
+        ),
+        SensorEntityDescription(
+            name="Duration in Traffic",
+            icon=ICONS.get(travel_mode, ICON_CAR),
+            key=ATTR_DURATION_IN_TRAFFIC,
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement=TIME_MINUTES,
+        ),
+        SensorEntityDescription(
+            name="Distance",
+            icon=ICONS.get(travel_mode, ICON_CAR),
+            key=ATTR_DISTANCE,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        SensorEntityDescription(
+            name="Route",
+            icon="mdi:directions",
+            key=ATTR_ROUTE,
+        ),
+        SensorEntityDescription(
```

Comment:
This shouldn't be a state, instead this should be `latitude` and `longitude` attributes so it can show up on a map too. As per your screenshot, it makes little sense

Reply:
I renamed `Origin Name` to `Origin` and `Destination Name` to `Destination` and added lat/lon as attributes